docs/content/en/providers/spotify.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Spotify
+description: This provider is based on oauth2 scheme and supports all scheme options
+position: 36
+category: Providers
+---
+
+[Source Code](https://github.com/nuxt-community/auth-module/blob/dev/src/providers/spotify.ts)
+
+⚠️ The Spotify provider is **still under development**, you can always use it in your local development.
```

Comment:
Is this still the current status? What needs to be complete before we take off this warning?

Reply:
No it should be ready for production. I will remove this warning.